pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -37,6 +38,18 @@ trait AuthActions extends PanDomainAuth {
    */
   def cacheValidation: Boolean = false
 
+
+  /**
+   * Adding an expiry extension to `APIAuthAction`s allows for a delay between an applications authentication and their
+   * respective API XHR calls expiring.
+   *
+   * This is particularly useful for SPAs where users have third party cookies disabled.
+   *
+   * @return the amount of delay between App and API expiry in milliseconds
+   */
+  def apiExpiryExtension: Long = 1.hour.toMillis
+
```

Comment:
Do we want to default this to an hour? might be better to have no grace period by default and make apps override this to enable the grace period.

Either way the doc comment should mention what the default behaviour is as well as what the setting does.